### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by tanvimoharir**
CODE_OF_CONDUCT file

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team at pranavmendi@gmail.com. All
```
Comment (Pranav016) [inline CODE_OF_CONDUCT.md:58]: use markdown and hide the email. (create a hyperlink that leads straight to the mail)
```
[Email](mailto:pranavmendi@gmail.com)
```

**Review by Pranav016 — CHANGES_REQUESTED** [review]
Also, squash your commits after working on the requested changes. Then you can force push to this branch.

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (tanvimoharir) [reply CODE_OF_CONDUCT.md]: This is changed.

**Comment by tanvimoharir** [comment]
@Pranav016  Please let me know in case of further changes or otherwise. Thanks.

**Comment by Pranav016** [comment]
> @Pranav016 Please let me know in case of further changes or otherwise. Thanks.

They are almost ready to be merged but I think you have increased your commits even more. (I have make sure the commit history stays clean) Its fine you are not able to squash but I'd recommend to make a new pull request with these files with just a single commit. That'll be great.

image

**Pull request closed without merging**